src/anshuKJun20/IntersectionOfArray.java

```diff
@@ -0,0 +1,45 @@
+package anshuKJun20;
+
+import java.util.Arrays;
+
+public class IntersectionOfArray {
+   
+	double[] getIntersectionArray(double[] arr1, double[] arr2) {
+		if(arr1.length<arr2.length) {
+			double[] tempArr = new double[arr1.length];
+		}else {
+			double[] tempArr = new double[arr2.length];
+		}
+		int count = 0;
+		for (int outerindex = 0; outerindex < arr2.length; outerindex++) {
+			
+			for (int innerindex = 0; innerindex < arr1.length; innerindex++) {
+				if (arr2[outerindex] == arr1[innerindex]) {
+					tempArr[count] = arr2[outerindex];
```

Comment:
I think here it should be a compilation error. Please verify.

Reply:
![image](https://user-images.githubusercontent.com/66829451/86517801-51a6d700-be49-11ea-8d08-06b4b218ddb5.png)
![Uploading image.png…]()
I checked it and also debug it. There is no any compilation error.

Reply:
![image](https://user-images.githubusercontent.com/66829451/86517844-bc581280-be49-11ea-854e-2c3a3ff3dc7c.png)